templates/hooks/secret/sdk_delete_post_build_request.go.tpl

```diff
@@ -0,0 +1,3 @@
+	if input.SecretId == nil {
+		input.SecretId = r.ko.Spec.Name
+	}
```

Comment:
This should already be done by the code generator

Reply:
removed in [remove redundant hooks](https://github.com/aws-controllers-k8s/secretsmanager-controller/pull/2/commits/a9b060b8117c8db81291c1142ca00f5ca4b59aa6)